docs/_providers/route53.md

```diff
@@ -24,13 +17,18 @@ It is also possible to specify an aws access key in the providers config json fi
 }
 {% endhighlight %}
 
-## Metadata
+You can also use environment variables, but this is discouraged.
```

Comment:
Please add back this link `[go sdk configuration](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)`

Reply:
Ok, just curious, what is the point of it?

Reply:
Some users configure it in different ways. If you are running from certain instance types, the environment might be implicitly set up, or they want to manage aws creds in the same way they manage other tools.

Reply:
Ok, I see now. I don't use that method myself, so I did not see the point. I'll add it.

Reply:
@captncraig @tlimoncelli Done.